### ISSUE TITLE
"Read More" Button Not Working

### ISSUE THREAD

**Issue by kaustubh0601** [vishal02527/My-Strength-Shree-Krishna#745]
**Describe the Bug**
Read More" button on text sections is not responding when clicked, preventing users from viewing the full content.

**Expected Behavior**
Initially, the text should be truncated to a specified limit. When the user clicks the "Read More" button, the full text should expand and become visible, allowing users to read the entire content.

**Screenshots**

https://github.com/user-attachments/assets/79c803a7-aebe-4025-aa39-8f9b717a9d32

**Comment by vishal02527** [comment]
@kaustubh0601 please complete your work as soon as possible

**Comment by vishal02527** [comment]
@kaustubh0601 Are you done with it?

**Comment by kaustubh0601** [comment]
@vishal02527  No. My task is done but i have face challenges in English Hindi toggling.

**Comment by eeshachavan** [comment]
Hello Vishal, I have solved this issue. The "read more" button for "Narsimha" is now working and taking us to narsi.html. 

However, I couldn't find the html pages of other avatars so i have only linked the page for Narasimha. Shall I push the changes as of now for you to check? 

![Image](https://github.com/user-attachments/assets/dc227b61-fd54-48d1-ac89-6a5ec526b517)
